features/step_definitions/wallets-steps.js

```diff
@@ -430,3 +430,15 @@ Then(/^I should see newly generated address as active address on the wallet rece
     return generatedAddress === activeAddress;
   });
 });
+
+Then(/^the wallets should be ordered from oldest to newest$/, async function () {
+
+  const [wallet1, wallet2, wallet3] = await this.client.getText('.SidebarWalletMenuItem_title');
```

Comment:
@daniloprates please update this test step to take the input values in a table form.